### PR TITLE
update illuminate up to 6.x

### PR DESCRIPTION
**Pull request by korobovn**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | No

### Changed

- Maximal `illuminate/*` packages version now is `6.*`

### Added

- GitHub actions for a tests running

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avto-dev/amqp-rabbit-manager/blob/master/CHANGELOG.md) file

### CODE REVIEW

**Review by tarampampam — CHANGES_REQUESTED** [review]
Remove `phpunit-without-external-services`

File: .github/workflows/execute-tests.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+      - name: Install basic Composer dependencies
+        if: matrix.setup == 'basic'
+        run: composer update --prefer-dist --no-interaction --no-suggest
+
+      - name: Show most important packages versions
+        run: composer info | grep -e laravel/laravel -e phpunit/phpunit -e phpstan/phpstan
+
+      - name: Execute tests
+        run: php ./vendor/bin/phpstan analyze --no-progress --ansi --level=max ./src && php ./vendor/bin/phpunit --no-coverage --exclude-group usesExternalServices
```
Comment (tarampampam) [inline .github/workflows/execute-tests.yml]: Make 2 actions:
- `Execute phpstan` (`composer phpstan`)
- `Execute phpunit` (`php ./vendor/bin/phpunit --no-coverage --exclude-group usesExternalServices`)

File: docker/app/Dockerfile [hunk 2]
```diff
@@ -35,11 +40,26 @@ RUN set -xe \
...
+    # Build openssl
+    && mkdir -p /usr/local/src && cd $_ \
+        && wget "https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz" -O "openssl-${OPENSSL_VERSION}.tar.gz" \
+        && echo "$OPENSSL_SHA256" "openssl-${OPENSSL_VERSION}.tar.gz" | sha256sum -c - \
+        && tar -zxvf "openssl-${OPENSSL_VERSION}.tar.gz" \
+        && cd "openssl-${OPENSSL_VERSION}" \
+        && ./config no-async shared --prefix=/usr/local/ssl --openssldir=/usr/local/ssl -Wl,-rpath,/usr/local/ssl/lib \
+        && make && make install \
+        && cp /usr/local/ssl/bin/openssl /usr/bin/openssl \
+        && rm -rf "/usr/local/src/openssl-${OPENSSL_VERSION}.tar.gz" "/usr/local/src/openssl-${OPENSSL_VERSION}"
```
Comment (tarampampam) [inline docker/app/Dockerfile]: This is required?

File: docker/app/Dockerfile [hunk 3]
```diff
@@ -35,11 +40,26 @@ RUN set -xe \
...
+
+#RUN ln -s /usr/local/ssl/lib/libcrypto.so.1.1 /usr/lib/libcrypto.so.1.1 \
+#        && ln -s /usr/local/ssl/lib/libssl.so.1.1 /usr/lib/libssl.so.1.1
+
     # this c-library is required for 'php-amqp'
```
Comment (tarampampam) [inline docker/app/Dockerfile]: Single `RUN` instruction broken

File: docker/app/Dockerfile [hunk 4]
```diff
@@ -35,11 +40,26 @@ RUN set -xe \
...
-    && ( git clone --branch v${RABBITMQ_VERSION} https://github.com/alanxz/rabbitmq-c.git /tmp/rabbitmq \
+RUN ( git clone --branch v${RABBITMQ_VERSION} https://github.com/alanxz/rabbitmq-c.git /tmp/rabbitmq \
         && cd /tmp/rabbitmq \
         && mkdir build && cd build \
-        && cmake .. \
+        && cmake -DCMAKE_C_FLAGS='-I/usr/local/ssl/include -L/usr/local/ssl/lib' \
+        -DOPENSSL_ROOT_DIR=/usr/local/ssl  -DOPENSSL_INCLUDE_DIR=/usr/local/ssl/include -DOPENSSL_LIBRARIES=/usr/lib .. \
```
Comment (tarampampam) [inline docker/app/Dockerfile]: This is required?

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: .github/workflows/execute-tests.yml (hunk 1 above)
Comment (korobovn) [reply .github/workflows/execute-tests.yml]: ok

File: docker/app/Dockerfile (hunk 2 above)
Comment (korobovn) [reply docker/app/Dockerfile]: for php:7.2.23 it isn't required. it was already removed (commented out)

File: docker/app/Dockerfile (hunk 3 above)
Comment (korobovn) [reply docker/app/Dockerfile]: fixed.
why is Single `RUN` necessary?
it doesn't allow use of the cache

File: docker/app/Dockerfile (hunk 4 above)
Comment (korobovn) [reply docker/app/Dockerfile]: I saved it as a comment

File: docker/app/Dockerfile (hunk 3 above)
Comment (tarampampam) [reply docker/app/Dockerfile]: Single `RUN` instruction creates **one** layer. Best practice - if you install any dependencies at the start and remove it them in a one `RUN` instruction

**Review by tarampampam — APPROVED** [review]

**Pull request merged**